.circleci/config.yml

```diff
@@ -96,13 +92,13 @@ jobs:
             cd ~/repo
 
             # Installs to move to image building
-            curl -O https://dl.google.com/go/go1.15.linux-amd64.tar.gz
-            tar -xvf go1.15.linux-amd64.tar.gz
+            curl -O https://dl.google.com/go/go1.15.linux-<< parameters.go_arch >>.tar.gz
```

Comment:
where are we using go and Hugo code in Hue?

Reply:
I am not sure about this.
I've just extracted the job definition into a CircleCI `command`, so that it is reusable for the jobs for the different CPU architectures (see https://github.com/cloudera/hue/pull/2556).

Reply:
Hi @ranade1, `Hugo` is used here for running documentation tests and `Go` is required for a repo `https://github.com/raviqqe/muffet` which check for dead links in the docs.